doc/manual/variables-and-scoping.rst

```diff
@@ -315,7 +473,7 @@ similar to ``while`` loops with ``let`` blocks inside::
     julia> Fs[2]()
     2
 
-``for`` loops will reuse existing variables for iteration::
+``for`` loops will reuse existing variables for its iteration variable::
```

Comment:
should this be: `variable(s)`? you can have a for loop with multiple iteration variables, i.e.
`for i=1:1000, j=1:20`